src/main/java/adris/altoclef/TaskCatalogue.java

```diff
@@ -194,7 +194,9 @@ public class TaskCatalogue {
                 // Don't mine individual planks either!! Handled internally.
                 woodCatalogue.dontMineIfPresent();
             }
-            shapedRecipe2x2("stick", Items.STICK, 4, p, o, p, o);
+            // shapedRecipe2x2("stick", Items.STICK, 4, p, o, p, o);
+            // mine("stick", MiningRequirement.HAND, Blocks.DEAD_BUSH, Items.STICK);
```

Comment:
you can remove the second comment since it doesn't indicate anything useful. 

the first one should be kept as it is helpful to know how we used to do this. That way if we want to make sticks more configurable we can.